spec.html

```diff
@@ -31640,14 +31640,17 @@ <h1>Notation</h1>
           <li>
             _Unicode_ is *true* if the RegExp object's [[OriginalFlags]] internal slot contains *"u"* and otherwise is *false*.
           </li>
+          <li oldids="sec-runtime-semantics-wordcharacters-abstract-operation">
+            _WordCharacters_ is the mathematical set that is the union of all sixty-three characters in *"ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789_"* (letters, numbers, and U+005F (LOW LINE) in the Unicode Basic Latin block) and all characters _c_ for which _c_ is not in that set but Canonicalize(_c_) is. _WordCharacters_ cannot contain more than sixty-three characters unless _Unicode_ and _IgnoreCase_ are both *true*.
+          </li>
         </ul>
         <p>Furthermore, the descriptions below use the following internal data structures:</p>
         <ul>
           <li>
-            A <em>CharSet</em> is a mathematical set of characters, either code units or code points depending up the state of the _Unicode_ flag. &ldquo;All characters&rdquo; means either all code unit values or all code point values also depending upon the state of _Unicode_.
+            A <em>CharSet</em> is a mathematical set of characters. &ldquo;All characters&rdquo; means the CharSet containing all code point values when the _Unicode_ flag is *true*, and otherwise means the CharSet containing all code unit values.
```

Comment:
Wordsmithing: I'd write this as

```suggestion
            A <em>CharSet</em> is a mathematical set of characters. When the _Unicode_ flag is *true* &ldquo;all characters&rdquo; means the CharSet containing all code point values; when it is *false* &ldquo;all characters&rdquo; means the CharSet containing all code unit values.
```

Reply:
Done.